test.js

```diff
@@ -34,4 +61,15 @@ function toArray (cb) {
       read(null, next) //loop again.
     })
   }
-}
\ No newline at end of file
+}
+
+function toArraySync(read){
```

Comment:
I've removed this as it was just a helper for left/right identity tests which have been removed